pridepy/tests/test_search.py

```
@@ -20,10 +20,8 @@ def test_search_projects(self):
         result = project.search_by_keywords_and_filters(keyword="PXD009476",query_filter="",
                                                         page_size=100,page=0 , sort_direction="DESC",
                                                         sort_fields="accession")
--        assert len(result) == 2
-+        # Verify that search returns at least one result and contains the expected project
-+        assert len(result) > 0, "Search should return at least one result"
-+        assert any(r["accession"] == "PXD009476" for r in result), "Search should return the queried project"
+        assert (len(result) > 0, "Search should return at least one result")
```

Ruff (0.8.2)

Assert test is a non-empty tuple, which is always `True`

(F631)